package.json

```diff
@@ -71,14 +71,25 @@
         "@carbon/icons": "^11.47.1",
         "@carbon/react": "^1.64.1",
         "@mavrin/remark-typograf": "^2.2.0",
-        "codemirror": "^5.65.16",
-        "codemirror-formatting": "^1.0.0",
+        "carbon-components-react": "^8.28.0",
+        "codemirror": "^6.0.1",
+        "@codemirror/legacy-modes": "^6.4.1",
+        "@codemirror/lang-markdown": "^6.2.5",
+        "@codemirror/lang-json": "^6.0.1",
+        "@codemirror/lang-xml": "^6.1.0",
+        "@codemirror/lint": "^6.8.1",
```

Comment:
Looks like `@codemirror/lint` is not used in gui elements code. Same for `@rdfjs/types`, `n3`, `@types/n3` and `xml-formatter`. Please remove all dependencies from `package.json` and `yarn.lock` if they are not necessary.